.circleci/verbatim-sources/header-section.yml

```diff
@@ -110,13 +110,8 @@ install_doc_push_script: &install_doc_push_script
 
     rm -rf pytorch || true
 
-    # Install TensorBoard so torch.utils.tensorboard classes render
-    PYTHON_VERSION=$(python -c 'import platform; print(platform.python_version())'|cut -c1)
-    if [[ $PYTHON_VERSION == "2" ]]; then
-      pip install -q https://s3.amazonaws.com/ossci-linux/wheels/tensorboard-1.14.0a0-py2-none-any.whl
-    else
-      pip install -q https://s3.amazonaws.com/ossci-linux/wheels/tensorboard-1.14.0a0-py3-none-any.whl
-    fi
+    # Install TensorBoard in python 3 so torch.utils.tensorboard classes render
```

Comment:
@lanpa would know, but I think these weren't built with universal in mind. That being said, this will be replaced with a `pip install tensorboard` in the future. The code we need hasn't gotten to the release channel yet, though.

Reply:
Sorry, I was asking about `.circleci/verbatim-sources/header-section.yml` vs`.circleci/.config.yml`

Reply:
Got it! So we have sections of yml in `/verbatim-sources/` that are combined programmatically when you run  `.circleci/regenerate.sh` (which @soumith pointed me at). I  think this is used to let us easily fanout across different CI versions like python2, python3, etc. but @kostmo added it initially I think.

Reply:
Ah that makes sense now. Thanks!